documentation/meta/release_and_deployment/index.md

```diff
@@ -0,0 +1,411 @@
+# The process of releasing and deploying code
+
+This document covers the code release and deployment processes for each service
+and environment. Colocating all information related to release and deployment
+(as opposed to separating them into the app-specific directories) allows us to
+more easily point out similarities and difference to keep in mind between each
+application, as these details tend to be where confusion happens the most.
```

Comment:
FWIW these lines, to me, read like motivation that would be (and is!) present in the PR description for this change rather than in the document itself. It could be removed, IMO.

Reply:
Commenting it out (so it can be retained but not displayed) sounds perfect IMO.